test/Datadog.Trace.Tests/SpanStatusTests.cs

```diff
@@ -0,0 +1,104 @@
+using Xunit;
```

Comment:
Taken from https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Tests/Trace/StatusTest.cs

Reply:
maybe I should mention it in a comment?

Reply:
done